### PR TITLE
Feature: Disable Notification Sound if not Available

### PR DESCRIPTION
**Pull request by ewth**
This adds functionality to disable notification sounds if status is not "Available".

If the user is in a meeting/call, or has set their status to "Busy", "Away", etc. - basically anything other than "Available" - then notification sounds won't be played.

### CODE REVIEW

File: app/browser/notifications/activityManager.js [hunk 1]
```diff
@@ -107,6 +108,7 @@ function meetingStartNotifyHandler(self) {
 // eslint-disable-next-line no-unused-vars
 function myStatusChangedHandler(self) {
 	return async (event) => {
+		self.ipcRenderer.send('user-status-changed', { data: event.data })
 		if (PresenceState.isAway(event, self)) {
```
Comment (jijojosephk) [inline app/browser/notifications/activityManager.js]: @ewth this was an experiment. Now they're no more required. 
```js
if (PresenceState.isAway(event, self)) {
			await evaluateAndPreventAwayStatus(self, event);
		}
```
If you think that's going to be difficult for you to remove. I can do that and you can merge your changes.

**Comment by jijojosephk** [comment]
@ewth Could you merge the changes and also bump the version in package.json?

File: app/browser/notifications/activityManager.js (hunk 1 above)
Comment (ewth) [reply app/browser/notifications/activityManager.js]: Thanks for the info @jijojosephk. I've removed it in a commit below (7e60447).

**Comment by jijojosephk** [comment]
@ewth  showing merge conflicts still.

**Comment by ewth** [comment]
@jijojosephk seems my fork fell behind, I'll close this PR, fix it up, and open another one (in case I can't get to it tonight).

**Pull request closed without merging**